src/client/components/Modal/Modal.component.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Modal.styles.css';
+import addedCart from '../../assets/images/addedCart.png';
+
+export default function ModalComponent({ show, handleLink, handleCloseModal }) {
+  if (show) {
+    return (
+      <div
+        style={{
+          zIndex: 999999,
+          position: 'fixed',
+          top: 0,
+          left: 0,
+          width: '100%',
+          height: '100vh',
+          backgroundColor: 'rgba(0,0,0,0.8)',
+        }}
+      >
+        <div
+          style={{
+            background: '#fff',
+            width: '30%',
+            padding: '1em',
+            margin: '10% auto',
+          }}
+        >
+          <div className="card-panel teal lighten-2 ">
+            <h2>ADDED TO THE CART</h2>
+            <img src={addedCart} alt="success" width="20%" />
+          </div>
+          <div>
+            <button type="button" onClick={handleLink} className="btnCart">
```

Comment:
There is the button component which you can use here, and that one should already have the correct styling

Reply:
Yes.. i did that .. but need to write new styling for other button .. thats why i used new buttons..